apps/web-giddh/src/app/ledger/ledger.component.ts

```diff
@@ -951,6 +961,43 @@ export class LedgerComponent implements OnInit, OnDestroy {
         }
     }
 
+    /**
+     * Prepare array and count the selected bank transaction to save.
+     *
+     * @param {*} [transaction]
+     * @memberof LedgerComponent
+     */
+    public getTransactionCountConvertToEntries(transaction?: any): void {
+        if (this.lc.bankTransactionsDebitData?.length || this.lc.bankTransactionsCreditData?.length) {
+            if (!transaction) {
+                let filteredDebit: any[] = [];
+                let filteredCredit: any[] = [];
+
+                this.lc.bankTransactionsDebitData.forEach(transaction => {
+                    if (transaction.transactions[0].selectedAccount?.name) {
+                        filteredDebit.push(transaction);
+                    }
+                });
+                this.lc.bankTransactionsCreditData.forEach(transaction => {
+                    if (transaction.transactions[0].selectedAccount?.name) {
+                        filteredCredit.push(transaction);
```

Comment:
don't create multiple array. Push in single array only

Reply:
DOne